src/examples/poi-crosschecker/poi-crosschecker.ts

```diff
@@ -82,23 +77,42 @@ const run = async () => {
         `\n📮 A new message has been received! Parse, validate, and store\n`
           .green
       );
-      const message = observer.readMessage(msg, NPOIMessage);
-      const attestation: Attestation = await observer.prepareAttestation(
-        message,
-        NPOIMessage
+      const message = await observer.readMessage({
+        msg,
+        topic,
+        types: [
+          { name: "subgraph", type: "string" },
+          { name: "nPOI", type: "string" },
+        ],
+      });
+
+      const { radioPayload, blockNumber, sender, stake } = message;
```

Comment:
maybe we could use a different name for `stake`, maybe something like `weight` or `score`? or we can add a comment that explains this field is for choosing the results in `sortAttestations`?

Reply:
I like `stakeWeight`, definition according to Google:

> Stake weight is a measurement of "staking power" each wallet that is staking has a stake weight equal to that of its available balance. The network stake-weight is the total number of all coins that are being staked on the entire network.

lmk how it sounds 👀

Reply:
Good point, I think `stakeWeight` is a good option, according to Google:
> Stake weight is a measurement of "staking power" each wallet that is staking has a stake weight equal to that of its available balance. The network stake-weight is the total number of all coins that are being staked on the entire network.

Obv that's related to wallets but in our case each `Attestation` has a `stakeWeight` that accumulates as messages come in, so it sounds fitting. Lmk if that makes sense 👀